bazel/repository_locations.bzl

```diff
@@ -14,42 +14,40 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "thrift-0.11.0",
     ),
     com_github_bombela_backward = dict(
-        commit = "44ae9609e860e3428cd057f7052e505b4819eb84",  # 2018-02-06
+        commit = "v1.4",
```

Comment:
use `tag` everywhere` for dependencies specified by a tag?

Reply:
I tried that but Bazel generates the following error:
```
ERROR: /home/moderation/Library/envoyproxy/envoy/WORKSPACE:6:1: Traceback (most recent call last):
        File "/home/moderation/Library/envoyproxy/envoy/WORKSPACE", line 6
                envoy_dependencies()
        File "/home/moderation/Library/envoyproxy/envoy/bazel/repositories.bzl", line 286, in envoy_dependencies
                _com_github_bombela_backward()
        File "/home/moderation/Library/envoyproxy/envoy/bazel/repositories.bzl", line 320, in _com_github_bombela_backward
                _repository_impl(name = "com_github_bombela_backw...", ...")
        File "/home/moderation/Library/envoyproxy/envoy/bazel/repositories.bzl", line 36, in _repository_impl
                fail(("Refusing to depend on Git tag ...)))
Refusing to depend on Git tag "v1.4" for external dependency "com_github_bombela_backward": use 'commit' instead.
ERROR: Error evaluating WORKSPACE file
ERROR: error loading package '': Encountered error while reading extension file 'bazel/repositories.bzl': no such package '@envoy_api//bazel': error loading package 'external': Could not load //external package
```